src/controllers/dashboard/plugins/installed/index.js

```diff
@@ -57,9 +77,9 @@ function getPluginCardHtml(plugin, pluginConfigurationPages) {
         html += '</div>';
     }
 
-    html += "<div class='cardText'>";
+    html += "<div class='cardText'>"
     html += configPage && configPage.DisplayName ? configPage.DisplayName : plugin.Name;
-    html += '</div>';
+    html += '<br/>Status: ' + plugin.Status +'</div>';
```

Comment:
Status should be translatable.

Reply:
Could point me in the direction of how this can be done?